precompiles/parachain-staking/StakingInterface.sol

```diff
@@ -73,11 +80,11 @@ interface ParachainStaking {
 // 	"8545c833": "is_candidate(address)",
 // 	"8f6d27c7": "is_selected_candidate(address)",
 // 	"8e5080e7": "is_nominator(address)",
-// 	"ad76ed5a": "join_candidates(uint256)",
-// 	"b7694219": "leave_candidates()",
-// 	"e8d68a37": "leave_nominators()",
+// 	"0a1bff60": "join_candidates(uint256,uint256)",
+// 	"72b02a31": "leave_candidates(uint256)",
+// 	"b71d2153": "leave_nominators(uint256)",
 // 	"c9f593b2": "min_nomination()",
-// 	"82f2c8df": "nominate(address,uint256)",
+// 	"49df6eb3": "nominate(address,uint256,uint256,uint256)",
```

Comment:
To review, calculate the Keccak256 of the function name on the right side value and make sure the first 8 digits of that hash match the left side value.

Reply:
changed tests to verify this